cognite/powerops/client/shop/shop_run_api.py

```diff
@@ -175,3 +197,50 @@ def retrieve(
             )
         else:
             raise TypeError(f"Invalid type {type(external_id)} for external_id.")
+
+    def retrieve_latest(
+        self,
+        watercourse: str | list[str] | None = None,
+        source: str | list[str] | None = None,
```

Comment:
This is causing a small issue. Since there is a method in the class `list` the class namespace will overwrite the built-on list, such that the list type hint here is actually the `list` function above. 

The solution is to move the `.list` method to the end of the class.